src/test/java/seedu/address/testutil/TestUtil.java

```diff
@@ -36,20 +36,20 @@ public static Path getFilePathInSandboxFolder(String fileName) {
      * Returns the middle index of the person in the {@code model}'s person list.
      */
     public static Index getMidIndex(Model model) {
-        return Index.fromOneBased(model.getFilteredPatientList().size() / 2);
+        return Index.fromOneBased(model.getFilteredDoctorList().size() / 2);
```

Comment:
some refactoring issues in this file, need to rewrite these methods since we now have 3 different things to `get` from